api/watcher/watcher_test.go

```diff
@@ -290,24 +291,25 @@ func (s *watcherSuite) assertSetupRelationStatusWatch(
 	// Create a macaroon for authorisation.
 	store, err := s.State.NewBakeryStorage()
 	c.Assert(err, jc.ErrorIsNil)
-	bakery, err := bakery.NewService(bakery.NewServiceParams{
-		Location: "juju model " + s.State.ModelUUID(),
-		Store:    store,
+	b := bakery.New(bakery.BakeryParams{
```

Comment:
(Suggestion), give the b object a better name.

Reply:
It's only used a couple of lines below and "bakery" clashes with the package name. "bkry" or something seems worse?